kube_scheduler/datadog_checks/kube_scheduler/data/conf.yaml.example

```diff
@@ -51,10 +51,19 @@ instances:
   - prometheus_url: http://localhost:10251/metrics
 
     ## @param leader_election - boolean - optional - default: true
-    ## Monitor the leader-election process on the kube-system:kube-scheduler endpoints.
+    ## Monitor the leader-election process on the kube-system:kube-scheduler.
     #
     # leader_election: true
 
+    ## @param leader_election_kind - string - optional - default: endpoints
+    ## Kind of object to look at for the leader election.
+    ## Can be:
+    ##   * lease
+    ##   * endpoints
+    ##   * configmap
+    #
+    # leader_election_kind: endpoints
+
```

Comment:
since `leader_election` is enabled by default, I'm wondering if the check should try the 3 possible kinds in case `leader_election_kind` was not configured

Reply:
Following this suggestion, I changed the default value of `leader_election_kind` to `auto`.
With this value, the check will try `lease` objects first. This is the new way.
If `lease` object isn’t available, it will fallback on `endpoints` object which is the legacy way.
https://github.com/DataDog/integrations-core/pull/8535/commits/837d76c0f1ec75904832f99b9d33599d26b4aa31